content/blog/2024/02/2024-02-26-jenkins-contributor-summit-and-fosdem-recap.adoc

```diff
@@ -0,0 +1,128 @@
+
+The return to in-person meant that not only could we gather in one location, but we were also able to collaborate and work together directly, something that is not normally possible due to the global spread of the Jenkins community.
+To make things even better, all of the Jenkins Officers and four of the five board members were able to travel to Brussels for the summit!
+
+Massive thanks to link:https://www.betacowork.com/[Betacowork] for providing a space that could hold the Contributor Summit.
+
+The day started with author:jmmeessen[Jean-Marc Meessen] providing an overview of the agenda.
+
+image::/images/post-images/2024/02/26/jean-marc-summit-intro.jpg[Jean-Marc Meessen provides an introduction for the Jenkins Contributor Summit.,width=839,height=631,align=center]
+
+author:markewaite[Mark Waite] then provided a review of the current state of Jenkins as a project.
+This covered everything from user and maintainer statistics to what the future of Jenkins will hopefully look like.
+
+image::/images/post-images/2024/02/26/mark-state-of-jenkins.jpg[Mark Waite provides an overview of the state of Jenkins.,width=839,height=631,align=center]
+
+Following Mark, the Jenkins SIG leaders and Officers provided insights into their various areas of knowledge.
+
+author:dduportal[Damien Duportal], the Infrastructure Officer, presented first, reviewing how Jenkins Infrastructure has evolved over the last year and what we are looking forward to in 2024.
+
+image::/images/post-images/2024/02/26/damien-infra-update.jpg[Damien Duportal provides an overview of the Jenkins infrastructure over 2023 and the goals for 2024.,width=839,height=631,align=center]
+
+Next up was author:timja[Tim Jacomb], the Release Officer, shared the successes and innovation that the project has experienced throughout 2023.
+
+image::/images/post-images/2024/02/26/tim-release-update.jpg[Tim Jacomb reviews the Jenkins release updates over 2023 and hopes for 2024.,width=839,height=631,align=center]
+
+After Tim wrapped up his section of the presentation, author:kmartens27[Kevin Martens], the Documentation Officer, shared what we hope 2024 will look like for Jenkins documentation and Jenkins.io.
+
+image::/images/post-images/2024/02/26/kevin-docs-update.jpg[Kevin Martens shares insight into what to look for in documentation for 2024.,width=839,height=631,align=center]
+
+As the Advocacy & Outreach SIG leader, author:alyssat[Alyssa Tong] then recapped all of the events that Jenkins held or participated in during 2023.
+
+image::/images/post-images/2024/02/26/alyssa-advocacy-update.jpg[Alyssa Tong recaps the events and successes of Jenkins' community.,width=839,height=631,align=center]
+
+She also shared the exciting news that Jenkins has recently been won the Most Innovative DevOps Open Source Project award from DevOps Dozen
+
+image::/images/post-images/2024/02/26/devops-award-group.jpg[Group photo of all the contributor summit attendees.,width=839,height=631,align=center]
```

Comment:
This image didn't display for me
![image](https://github.com/jenkins-infra/jenkins.io/assets/101582594/564db77a-d3cf-4a59-97e6-3c85a83190e5)